libs/plugin-cache/src/lib/store-adapters/storage-adapter.ts

```diff
@@ -3,4 +3,5 @@ import { Observable } from 'rxjs';
 export interface StorageAdapter {
   get(key: string): Observable<string>;
   set(key: string, value: string): void;
+  unset(key: string): void;
```

Comment:
what about `remove`?

Reply:
I just noticed you switched to `delete` but I feel like `remove` would be better as `delete` is a js keyword.